### PR TITLE
New Styles structure

### PR DESCRIPTION
**Pull request by DianaRenz**

### CODE REVIEW

File: portal/code/cx-portal/src/components/styles/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: FabricMDL2Icons-0;
+  src: url(https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/10.0.0/css/fabric.min.css);
```
Comment (oyo) [inline portal/code/cx-portal/src/components/styles/_fonts.scss]: are we sure this is a reliable source?

**Review by oyo — APPROVED** [review]
approved - nur die Frage wegen dieser URL zum css. Aber es sieht so aus als wenn das von MS gehostet ist. Ich denke das ist okay.

File: portal/code/cx-portal/src/components/styles/_fonts.scss (hunk 1 above)
Comment (DianaRenz) [reply portal/code/cx-portal/src/components/styles/_fonts.scss]: I'm not sure. I just copy/paste it from main.scss
Do we need it generally? Or can I delete it? I don't see, that we ever used it

Comment (oyo) [reply portal/code/cx-portal/src/components/styles/_fonts.scss]: I don't know where this came from. Maybe when I originally copied some styles from the old portal. I think you can delete that.

**Pull request merged**